manifests/download.pp

```diff
@@ -0,0 +1,64 @@
+# == Definition: archive::download
```

Comment:
sorry, I wasn't clear (sorry of my life!) I don't mean document all the parameters etc.  I mean just add a line to say that it is a shim/private etc.  Something to discourage new users from thinking this is the way they should be using the module.